src/Common/src/System/Security/Cryptography/Asn1Reader/AsnReader.Integer.cs

```diff
@@ -107,9 +107,7 @@ public BigInteger ReadInteger(Asn1Tag expectedTag)
             }
             finally
             {
-                // Clear the whole tmp so that not even the sign bit is returned to the array pool.
```

Comment:
Presumably the comment was here to prevent some future dev from "optimizing" this?  Is it worth keeping for that reason?